<launch>
        <arg name="config_path" default = "$(find itri)/config/initial.yaml" />

        <arg name="rviz" default="false"/>
            <group if="$(arg rviz)">
                <param name="use_sim_time" value="true"/>
                <node pkg="rviz" type="rviz" name="rviz" args="-d $(find itri)/rviz_cfg/rviz.rviz" />
            </group>

        <arg name="feature" default="true"/>
        <param name="config_file" type="string" value="$(arg config_path)" />
        <node pkg="itri" type="initial" name="initial" launch-prefix="taskset -c 3" output="screen"/>
</launch>

